httpx/utils.py

```diff
@@ -94,8 +95,9 @@ def guess_json_utf(data: bytes) -> typing.Optional[str]:
 NETRC_STATIC_FILES = (Path("~/.netrc"), Path("~/_netrc"))
 
 
-def get_netrc_login(host: str) -> typing.Optional[typing.Tuple[str, str, str]]:
-    NETRC_FILES = (Path(os.getenv("NETRC", "")),) + NETRC_STATIC_FILES
+@functools.lru_cache()
+def get_netrc(netrc_env: str) -> typing.Optional[typing.Any]:
```

Comment:
Instead of `typing.Any` let's have `typing.Optional[netrc.netrc]` because `netrc` is actually a class.

Reply:
Actually I tried as netrc but I couldn't :) I will try again